cmd/benchmark.go

```diff
@@ -15,6 +15,14 @@ import (
 func main() {
 	runs := flag.Int("runs", 100, "number of runs per benchmark")
 	profile := flag.String("profile", "benchmark", "profile to use for minikube commands")
+	// images is a list of images we use, seperated by ,
+	images := flag.String("images", "", "a list of images we use, seperated by comma")
+
+	iters := flag.String("iters", "iterative,non-iterative", "a list of flows we use, seperated by comma")
+
+	benchMethods := flag.String("bench-methods", "", "a list of benchmark method names we use,seperated by comma")
```

Comment:
```suggestion
	benchMethods := flag.String("bench-methods", "", "a comma separated list of benchmark method names")
```

Could you include an example value in the description as well